src/Calculator/App.xaml

```diff
@@ -1419,103 +1422,74 @@
-
                                     <VisualStateGroup x:Name="ButtonStates">
                                         <VisualState x:Name="ButtonVisible">
-                                            <Storyboard>
-                                                <ObjectAnimationUsingKeyFrames Storyboard.TargetName="DeleteButton" Storyboard.TargetProperty="Visibility">
-                                                    <DiscreteObjectKeyFrame KeyTime="0">
-                                                        <DiscreteObjectKeyFrame.Value>
-                                                            <Visibility>Visible</Visibility>
-                                                        </DiscreteObjectKeyFrame.Value>
-                                                    </DiscreteObjectKeyFrame>
-                                                </ObjectAnimationUsingKeyFrames>
-                                            </Storyboard>
+                                            <VisualState.Setters>
+                                                <Setter Target="DeleteButton.Visibility" Value="Visible"/>
+                                            </VisualState.Setters>
                                         </VisualState>
                                         <VisualState x:Name="ButtonCollapsed"/>
                                     </VisualStateGroup>
-
+                                    <VisualStateGroup x:Name="AddEquationModeStates">
+                                        <VisualState x:Name="AddEquationModeState">
+                                            <VisualState.Setters>
+                                                <Setter Target="EquationButton.Visibility" Value="Collapsed"/>
+                                                <Setter Target="RemoveButtonPanel.Visibility" Value="Collapsed"/>
+                                                <Setter Target="EquationTextBoxAddPanel.Visibility" Value="Visible"/>
+                                                <Setter Target="EquationBoxBackground.Opacity" Value="{ThemeResource EquationBoxEquationAddPanelOpacity}"/>
+                                            </VisualState.Setters>
+                                        </VisualState>
+                                        <VisualState x:Name="EquationModeState"/>
+                                    </VisualStateGroup>
                                 </VisualStateManager.VisualStateGroups>
 
+                                <Border x:Name="EquationTextBoxAddPanel"
+                                        x:Uid="EquationTextBoxAddPanel"
+                                        MinWidth="44"
+                                        MinHeight="44"
+                                        Background="{TemplateBinding EquationColor}"
+                                        Visibility="Collapsed">
+                                    <FontIcon Foreground="{StaticResource SystemChromeWhiteColor}"
```

Comment:
This should be SystemColorButtonTextColor in high contrast modes

Reply:
Marco's PR will add new brushes, including one that will be used for this icon. Let's keep it like that for the moment, we will modify that after the other PR merged.